### PR TITLE
DSA-17-maximum-product-subarray

### PR DESCRIPTION
**Pull request by charan-2316**
It is  used to find the product of the maximum product subarray.

### CODE REVIEW

File: DSA_Training/MaximumProductSubarray.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
```
Comment (ganeshyevle) [inline DSA_Training/MaximumProductSubarray.java]: Please, Add comments here

**Review by ganeshyevle — CHANGES_REQUESTED** [review]
Please, add Comments

File: DSA_Training/MaximumProductSubarray.java (hunk 1 above)
Comment (charan-2316) [reply DSA_Training/MaximumProductSubarray.java]: done

**Review by charan-2316 — COMMENTED** [review]
done

**Review by ganeshyevle — APPROVED** [review]

**Pull request merged**